datastore/tasks.js

```diff
@@ -152,101 +158,52 @@ function deleteTask (taskId, callback) {
     taskId
   ]);
 
-  datastore.delete(taskKey, callback);
+  datastore.delete(taskKey, function (err) {
+    if (err) {
+      return callback(err);
+    }
+
+    return callback();
```

Comment:
Add the following above this line: `console.log('Task %d deleted successfully.', taskId);`